dom.bs

```diff
@@ -2440,7 +2444,8 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
    <li><p>Otherwise, <a for=set>insert</a> <var>node</var> into <var>parent</var>'s
    <a for=tree>children</a> before <var>child</var>'s <a for=tree>index</a>.
 
-   <li><p>If <var>parent</var> is a <a for=Element>shadow host</a> and <var>node</var> is a
+   <li><p>If <var>parent</var> is a <a for=Element>shadow host</a> with its <a for=tree>root</a>'s
```

Comment:
Don't you mean parent's shadow root here? Parent's root might well be a document or something else.

Reply:
Right! Parent's shadow root.  Good to know I had to make this super clear.